new-frontiers-14/frontier-station-14
# Restore missing Traits Postgres migration

## About the PR



Adds the upstream Traits table Postgres migration from August 2022.

## Why / Balance


The Frontier BankBalance migration fails to run because the index it removes was added in this missing migration.  This should fix the issue.

## How to test

1. Build the Frontier server, point it at a clean, uninitialized Postgres database.
2. The server should initialize its database and start without issue.
3. Connect to the server, you should connect without issue.
4. The server should run.

## Media


## Requirements



## Breaking changes


**Changelog**


No changelog, internal feature.
